.github/workflows/release-catalog.yaml

```diff
@@ -55,5 +55,8 @@ jobs:
           registry: quay.io
           username: ${{ secrets.REGISTRY_USER }}
 
-      - name: Build and push bundle container image
+      - name: Build and push the catalog container image
         run: make catalog-push
+
+      - name: Build and push the current catalog container image as latest
+        run: make catalog-retag-latest
```

Comment:
Instead of doing 2 executions: `catalog-push` and `catalog-retag-latest`

Right now is documented for alpha releases https://github.com/3scale-ops/prometheus-exporter-operator/blob/main/docs/release.md#alpha to do a single execution called `release-publish`, which seems to do everything, isn't it? 

```
catalog-publish: catalog-add-bundle catalog-build catalog-push catalog-retag-latest ## Builds and pushes the catalog image

release-publish: container-push bundle-push catalog-publish ## Publish a new stable release (operator, catalog and bundle)
```

Reply:
Yes, buy the container and the bundle in build and pushed in the `release.yaml` workflow. Using the release in the `release-catalog` workflow, will rebuild and repush them. 

In the alpha releases, as the build is local, can be done in a single target. For stables, it needs to be done in 2 steps.